cmake/mfmgConfig.cmake.in

```diff
@@ -4,3 +4,9 @@ SET(MFMG_VERSION_PATCH @MFMG_VERSION_PATCH@)
 SET(MFMG_VERSION @MFMG_VERSION@)
 
 SET(MFMG_BUILD_TYPE @CMAKE_BUILD_TYPE@)
+
+GET_FILENAME_COMPONENT(mfmg_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+
+IF(NOT TARGET mfmg::mfmg)
+  INCLUDE("${mfmg_CMAKE_DIR}/mfmgTargets.cmake")
```

Comment:
Do we have a file `mfmgTargets.cmake` or is it automatically generated?

Reply:
That file is generated via https://github.com/ORNL-CEES/mfmg/pull/214/files#diff-af3b638bc2a3e6c650974192a53c7291R131.